doc/classes/MeshInstance3D.xml

```diff
@@ -20,6 +20,13 @@
 				This helper creates a [StaticBody3D] child node with a [ConvexPolygonShape3D] collision shape calculated from the mesh geometry. It's mainly used for testing.
 			</description>
 		</method>
+		<method name="create_multiple_convex_collision">
```

Comment:
Shouldn't it be plural "collisions"?

Reply:
Possibly, but this is consistent with the other 2 functions. Would you like me to change it?

Reply:
If "collision" stands for "collision shape", then yeah this method should likely have a plural since it creates multiple collision shapes (i.e. multiple "collisions").